python/mxnet/gluon/block.py

```diff
@@ -1355,9 +1359,18 @@ def export(self, path, epoch=0, remove_amp_cast=True):
                     else:
                         arg_dict['aux:%s'%name] = param._reduce()
         save_fn = _mx_npx.save if is_np_array() else ndarray.save
-        params_filename = '%s-%04d.params'%(path, epoch)
-        save_fn(params_filename, arg_dict)
-        return (sym_filename, params_filename)
```

Comment:
is there any reason to preserve old behavior when actually exporting to a file?

Reply:
Do you mean remove the `aux:` and `arg:` prefixes. I'm fine with that

Reply:
no i meant `return (sym_filename, params_filename)`

Reply:
I think it can still be useful, for example to print the filenames.

Reply:
cool, looks like you're already doing that! 👍